misk/src/main/kotlin/misk/web/actions/WebActionMetadataAction.kt

```diff
@@ -30,6 +31,44 @@ class WebActionMetadataAction @Inject constructor() : WebAction {
   }
 
   data class Response(val webActionMetadata: List<WebActionMetadata>)
+
+  class Factory {
```

Comment:
rather than an additional layer of abstraction let's have a top level method on the action